scripts/install.sh

```diff
@@ -151,10 +179,10 @@ install_app() {
     tar -xzf "/tmp/$ASSET_NAME" -C "$HOME/.local"
 
     # --- remove tarball after installation
-    rm "/tmp/$ASSET_NAME"
+    find "/tmp" -maxdepth 1 -type f -name "$ASSET_NAME" -exec bash -c 'echo "File {} Removed"; rm -rf {}' \;
```

Comment:
The current approach seems complex for this task. Since the goal is to remove a single file or folder, using `find` is overkill and may make the code harder to read.

I suggest refactoring it to use a straightforward function, something like this:

```bash
remove_if_exists() {
    local target="$1"
    
    if [ -z "$target" ]; then
        logger "No target specified in remove_if_exists function"
        return 1
    fi

    if [ -e "$target" ]; then
        logger "File \"$target\" Removed"
        rm -rf "$target"
    else
        logger "File \"$target\" does not exist"
    fi
}